scripts/Mansion.gd

```diff
@@ -118,6 +118,19 @@ func _ready():
 	_on_mansion_pressed()
 	#startending()
 
+var awayReturnText = {
+	'travel back': 'returned to the mansion and went back to $his duty.',
```

Comment:
Does it sound slightly better if we say "back to $his duties"? I'm not a native speaker, but I believe it's usually plural.

Reply:
You are correct, it is typically pluralized